jazon-core/src/main/java/com/zendesk/jazon/actual/ActualJsonNumber.java

```diff
@@ -56,4 +56,13 @@ private static boolean isOfSupportedType(Number number) {
                 number instanceof Float ||
                 number instanceof Double;
     }
+
+    private Number maybeConvertLongToInt(Number number) {
```

Comment:
@marcintuleja As we talked offline, maybe `sanitized()` as this method name suits better?

Reply:
@pawel-mikolajczyk yup, `sanitize()` seems cool (maybe with a short comment why we want to do this)